appdev/guides/user-metrics.rst

```diff
@@ -77,17 +89,29 @@ User metrics are made up of two "formats"
 - metrics/messages for today (``format``)
 - metrics/messages for tomorrow (``emptyFormat``)
 
-The value of ``emptyFormat`` is what displays on the lock screen when no value has been stored in ``format``. In order to display a new value of ``format`` the metric must be updated with ``metric.update(x)`` (where x is the number of milliseconds until the update takes place).
+The value of ``emptyFormat`` is what displays on the lock screen when no value has been stored in ``format``. In order to display a new value of ``format`` the metric must be updated.
+
+Methods for the metric component (`source <https://daker.me/2013/11/adding-usermetrics-to-your-app-on-ubuntu-touch.html>`_):
+
```

Comment:
This line looks a bit odd https://ci.ubports.com/job/docs.ubports.com/job/PR-290/1/artifact/_build/html/appdev/guides/user-metrics.html

Maybe you meant this to be a new subsection heading? It would need to be formatted as such. However, I actually think we don't need to start a new subsection here, so I would suggest to remove this line entirely. 

The linked blog post is useful, but calling it "source" and sticking it at the end of this label, I think is actually confusing. I'd suggest adding a sentence at the bottom of the section, or the bottom of the page. Smt like:

"See this blog post for a simple example."

and put the link behind example ... Or smt like that